mopidy_beets/__init__.py

```diff
@@ -1,29 +1,28 @@
-from __future__ import unicode_literals
-
 import os
 
 from mopidy import config, ext
 
 
-__version__ = '3.1.0'
+__version__ = "3.1.0"
```

Comment:
To not duplicate the version tag:

```suggestion
__version__ = pkg_resources.get_distribution("Mopidy-Beets").version
```

Reply:
Interesting - I did not know this approach.

But I noticed, that this breaks local tests of a git checkout (`pkg_resources.DistributionNotFound: The 'Mopidy' distribution was not found and is required by the application`). Is there a way around this?

Anyway: I introduced a build dependency on `bumpversion`. Thus I would not mind an additional version string to be updated automatically.

Reply:
Ah, yes. It should work if you install it with pip in "editable" mode in the virtualenv where you do your development, ref. https://docs.mopidy.com/en/develop/devenv/#install-mopidy-from-the-git-repo